addons/delivery/models/delivery_carrier.py

```diff
@@ -240,6 +240,10 @@ def rate_shipment(self, order):
                 res['warning_message'] = _('The shipping is free since the order amount exceeds %.2f.', self.amount)
                 res['price'] = 0.0
             return res
+        else:
+            return {
+                'error_message': _("Shipment rate not found")
```

Comment:
Don't we need to export pot file then? Or, in this case, maybe just remove the `_()`?